lib/components/MenuRenderer/MenuRenderer.css.ts

```diff
@@ -11,3 +12,18 @@ export const menuIsClosed = style({
   transform: `translateY(${calc(vars.grid).negate().multiply(2)})`,
   visibility: 'hidden',
 });
+
+const widthVar = createVar();
+const baseWidth = style({
+  width: calc(widthVar).divide(4).toString(),
+});
+
+const widths = omit(vars.contentWidth, 'xsmall', 'xlarge');
+export const width = styleVariants(widths, (w) => [
```

Comment:
That would result in unused variables, but looking at this again, i think it should be more of an opt in for widths, rather than opt out.
```suggestion
const { small, medium, large } = vars.contentWidth;
export const width = styleVariants({ small, medium, large }, (w) => [
```